src/test/java/com/easypost/TestUtils.java

```diff
@@ -0,0 +1,206 @@
+        add("Authorization");
+        add("X-Client-User-Agent");
+        add("User-Agent");
+    }};
+
+    private static final List<String> QUERY_CENSORS = new ArrayList<String>() {{
+        add("card[number]");
+        add("card[cvc]");
+    }};
+
+    private static final List<String> BODY_CENSORS = new ArrayList<String>() {{
+        add("api_keys");
+        // add("children");
+        add("client_ip");
+        add("credentials");
+        add("key");
+        add("keys");
+        add("phone_number");
+        add("phone");
+        add("test_credentials");
+        // add("created_at"); TODO: Add as ignore in body in future
+        // add("updated_at");
+    }};
+
+    private static String getSourceFileDirectory() {
+        try {
+            return Paths.get("").toAbsolutePath().toString();
+        } catch (Exception e) {
+            return "";
+        }
+    }
+
+    /**
+     * Get an API key from the environment.
+     * @param apiKey Type of API key to get
+     * @return the API key from the environment
+     */
+    public static String getApiKey(ApiKey apiKey) {
+        String keyName = "";
+        switch (apiKey) {
+            case TEST:
+                keyName = "EASYPOST_TEST_API_KEY";
+                break;
+            case PRODUCTION:
+                keyName = "EASYPOST_PROD_API_KEY";
+                break;
+            default:
+                break;
+        }
+
+        String value = System.getenv(keyName);
+        return value != null ? value :
+                API_KEY_FAILED_TO_PULL; // if can't pull from environment, will use a fake key. Won't matter on replay.
+    }
+
+    public enum ApiKey {
+        TEST,
+        PRODUCTION
+    }
+
+    public static final class VCR {
+        private final com.easypost.easyvcr.VCR vcr;
+
+        private final String apiKey;
+
+        private String testCassettesFolder;
+
+        /**
+         * Get whether the VCR is recording.
+         *
+         * @return true if recording, false otherwise.
+         */
+        public boolean isRecording() {
+            return vcr.getMode() == Mode.Record;
+        }
+
+        /**
+         * Constructor.
+         *
+         * @param testCassettesFolder The folder where the cassettes will be stored.
+         * @param apiKey              The API key to use.
+         */
+        public VCR(String testCassettesFolder, String apiKey) {
+            AdvancedSettings advancedSettings = new AdvancedSettings();
+            advancedSettings.matchRules = MatchRules.regular(); // match by method, url
+            // TODO: Change to strict when VCR updated with ignore aspect
+            advancedSettings.censors =
+                    new Censors("<REDACTED>").hideHeaders(HEADER_CENSORS).hideQueryParameters(QUERY_CENSORS)
+                            .hideBodyParameters(BODY_CENSORS);
+            advancedSettings.simulateDelay = false;
+            advancedSettings.manualDelay = 0;
```

Comment:
Do we need these or will VCR set these defaults for us? If so, let's remove them here